spec/package_spec.rb

```diff
@@ -372,16 +372,6 @@ def pages_dir_nil
     expect(Dir.exist?(directory)).to be_falsey
   end
 
-  def mock_onesky(project_id = nil)
-    ENV["ONESKY_API_SECRET"] ||= ""
-    project_id ||= Resource.find(1).onesky_project_id
-    response = RestClient::Response.new('{ "1":"value" }')
-    response.instance_variable_set :@code, 200
-    allow(RestClient).to receive(:get)
-      .with("https://platform.api.onesky.io/1/projects/#{project_id}/translations", any_args)
-      .and_return(response)
-  end
```

Comment:
I'm curious about this mock being removed, none of this work should have touched OneSky interactions (other than them being transitively triggered within the publish process which now fires from sidekiq

Reply:
nm, I see it was moved to a different location